package.json

```diff
@@ -19,5 +19,8 @@
   "engines": {
     "yarn": "^1.22.10",
     "node": "^14.15"
+  },
+  "dependencies": {
+    "connect-flash": "^0.1.1"
```

Comment:
interestingly, new dependencies should be added to either `server/package.json` or `client/package.son` depending on which environment they are used in (even if they are used in both environments, in which case both should be updated to have the dependency. 

I also dont see where this package is being used. It is a server side library. If you have a react frontend, then React should manage displaying the flash message. Though you could store your message server side in a session, you would need a subsequent request to also grab that flash message and retrieve it.

Much like `currentUser` state, you could also store a message in state of App.js and access it from there after redirecting a user.

Reply:
We were going to use that with an authentication issue, but we no longer are using it.  

The files have been deleted